schema-supplement.rst

```diff
@@ -117,9 +100,9 @@ Specifies the URI at which the package's website may be found.
 
 .. ... .. ... .. ... .. ... .. ... .. ... .. ... .. ... .. ... .. ... .. ... ..
 
-.. _SPDX: https://spdx.org/
+.. _SPDX: https://spdx.org/licenses/
```

Comment:
Given these references are referring to the organization itself, I'm inclined to leave them pointing to the root page. Also, if we're being pedantic, the canonical license list is Annex A, which is linked via Annex D (i.e. our "license expressions" link). Anyway, with these changes, we don't actually call out the license *list* specifically.

```suggestion
.. _SPDX: https://spdx.org/
```

...but I'll consider counter-arguments. :slightly_smiling_face: